canonical-websites/www.ubuntu.com
# Flattened legal managed services section

## Done

- Flattened legal managed services section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/managed-services-service-description](http://0.0.0.0:8001/legal/managed-services-service-description) and [the old link](http://0.0.0.0:8001/legal/managed-services/service-description) redirects also test that [/mssd](http://0.0.0.0:8001/mssd) redirects as well
* test the link [/legal/managed-services-privacy](http://0.0.0.0:8001/legal/managed-services-privacy) and [the old link](http://0.0.0.0:8001/legal/managed-services/privacy) redirects


## Issue / Card

Fixes #4195
